From 1065948945d4ff475f6db1b294c2e024a934561c Mon Sep 17 00:00:00 2001
From: Newbyte <newbyte@postmarketos.org>
Date: Thu, 7 Aug 2025 11:47:12 +0200
Subject: [PATCH] Metainfo fixes (#2939)

* Metainfo: Add 0.21.0

* Metainfo: Replace <developer_name> with <developer>

The developer_name tag is deprecated[1].

 [1]: https://freedesktop.org/software/appstream/docs/chap-Validation.html#asv-developer-name-tag-deprecated
---
 zola.metainfo.xml | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/zola.metainfo.xml b/zola.metainfo.xml
index 2f60625a82..ddf1d94a9f 100644
--- a/zola.metainfo.xml
+++ b/zola.metainfo.xml
@@ -5,7 +5,9 @@
 
   <name>Zola</name>
   <summary>A fast static site generator in a single binary with everything built-in</summary>
-  <developer_name>Vincent Prouillet and contributors</developer_name>
+  <developer>
+    <name>Vincent Prouillet and contributors</name>
+  </developer>
 
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>MIT</project_license>
@@ -51,6 +53,9 @@
   <content_rating type="oars-1.1" />
 
   <releases>
+    <release version="0.21.0" date="2025-07-14">
+      <url>https://github.com/getzola/zola/releases/tag/v0.21.0</url>
+    </release>
     <release version="0.20.0" date="2025-02-15">
       <url>https://github.com/getzola/zola/releases/tag/v0.20.0</url>
     </release>
